device-identifier/shbdeviceidentifier/db.py

```diff
@@ -154,11 +248,6 @@ def _setup_SQLite_db(self):
                                                id integer PRIMARY KEY,
                                                username VARCHAR NOT NULL UNIQUE
                                            );"""
-
-        sql_add_user = """INSERT OR IGNORE INTO users (username)
```

Comment:
Why is this part missing? When I test I cannot find any users in `users` anymore.

Reply:
Fixed